ultra/competition/starting_kit/README.md

```diff
@@ -0,0 +1,372 @@
+# Starting Kit
+
+Welcome to the ULTRA competition! This README provides instructions on how to get started by installing all the dependencies and running a quick example. Additionally, it will walk through the basics of creating your own agent, creating your own scenarios, and submitting your agent for evaluation.
+
+For more comprehensive documentation on SMARTS and ULTRA, please visit these links:
+- https://smarts.readthedocs.io/
+- https://github.com/huawei-noah/SMARTS/tree/ultra-develop/ultra/docs
+
+## Setup
+If not available, install Docker from:
+https://docs.docker.com/get-docker/.
+
+Build and run Docker container:
+```bash
+$ cd /path/to/SMARTS/ultra/competition/starting_kit
+$ docker build --no-cache --network=host -f Dockerfile -t starting_kit:latest .
+$ docker run --rm -it --network=host -v ${PWD}:/starting_kit starting_kit:latest
+
+# Inside the Docker container
+(docker) $ cd /starting_kit
```

Comment:
```suggestion
$ cd /starting_kit
```

Reply:
The `(docker)` prefix was to reiterate that the command is to be executed inside the docker container.